tests/e2e/standard/test_cli.py

```diff
@@ -114,7 +115,10 @@ def test_sync_container(self, container_type, environment):
 
     @staticmethod
     def stop_synchronization_process(container):
-        container._op_processor._operation_processor._consumer.interrupt()
+        if isinstance(container._op_processor, LazyOperationProcessorWrapper):
+            container._op_processor._operation_processor._consumer.interrupt()
```

Comment:
Unreachable, right? Why not just `container._op_processor._consumer.interrupt()`?

Reply:
Based on the tests above it's always `AsyncOperationProcessor` if I understand

Reply:
Well in this case it is `AsyncOperationProcessor` but I wanted to include case when it's wrapped with `LazyOperationProcessorWrapper`

Reply:
Nah, I don't think it's worth it, and it introduces some friction.